### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by UlisesGascon**
### Changelog

*Documentation #21*

- https://github.com/terrierscript/gulp-unzip/commit/12499b6aed12022e6bac7434a43477ed6709750d Test migration to ES6
- https://github.com/terrierscript/gulp-unzip/commit/aab0172386c68927afba1be08bb2bb5bc1a555c5Core migration to ES6
- https://github.com/terrierscript/gulp-unzip/commit/9c6707fd81d3324c9adf45382fe3eaa796e20fdb Dependencies updated
    - `through2` from `^0.4.1` to `3.0.0` 
    - `unzipper` from `^0.7.1` to `0.9.10` 
    - `mocha` from `^1.18.0` to `5.2.0`
    - `assert-plus` from `^0.1.5` to `1.0.0`
    - `minimatch` from `^0.2.14` to `3.0.4`
- https://github.com/terrierscript/gulp-unzip/commit/9c6707fd81d3324c9adf45382fe3eaa796e20fdb Dependencies removed
    - `defaults`
    - `fancy-log`
- https://github.com/terrierscript/gulp-unzip/commit/ed79df4313aeb9148a7553ff7b8171c8f3fe70b1 Added `CONTRIBUTING.md`
- https://github.com/terrierscript/gulp-unzip/commit/4ca7b1655ad7c2e89bcc047f46cebd1c59ba1158 Added `LICENSE`
- https://github.com/terrierscript/gulp-unzip/commit/64ecb3fd74a072e95971dcfebe938e0cb5320a10 Improved documentation

### CODE REVIEW

**Comment by UlisesGascon** [comment]
@terrierscript let me know what do you think about the changes... and if you agree let's merge it and  publish the new release to NPM ;-)

File: package.json [hunk 1]
```diff
@@ -16,19 +16,47 @@
     "zip",
     "package"
   ],
-  "author": "inuscript",
+  "author": {
+    "name": "S. Suzuki",
```
Comment (terrierscript) [inline package.json]:
```suggestion
    "name": "terrierscript",
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,11 @@
+The BSD 2-Clause License
+Copyright © 2014 S. Suzuki
```
Comment (terrierscript) [inline LICENSE]:
```suggestion
Copyright © 2014 terrierscript
```

File: README.md [hunk 3]
```diff
@@ -35,26 +46,33 @@ gulp.task('filter_sample', function(){
...
 ```
 
-# Entry
+### Contributors
+
+If you want to contribute to the project, please check de [Contribution Guidelines](CONTRIBUTING.md)
 
-For more info, go to [node-unzip](https://github.com/EvanOxfeld/node-unzip).
+**Author**
+- [S. Suzuki (@terrierscript)](https://github.com/terrierscript)
```
Comment (terrierscript) [inline README.md]:
```suggestion
- [terrierscript (@terrierscript)](https://github.com/terrierscript)
```

**Comment by UlisesGascon** [comment]
Awesome! suggestions accepted 👍

**Comment by terrierscript** [comment]
I published it

**Comment by UlisesGascon** [comment]
Awesome! Thanks @terrierscript !!

**Pull request merged**